dev-docs/bidders/aduptech.md

```diff
@@ -4,6 +4,7 @@ title: AdUp Technology
 description: Prebid Bidder Adapter for AdUp Technology
 biddercode: aduptech
 gdpr_supported: true
+floors_supported: true
 gvl_id: 647
 media_types: banner, native
 pbjs: true
```

Comment:
Thanks for updating the docs. Could you add the flag multiformat_supported (introduced in #3694 ) as well?

It allows sales houses / publishers to better understand the behaviour of the bid adapter for multi format ad units.